internal/admin/keys_access.go

```
@@ -0,0 +1,85 @@
+var viewerPersonalProviders = map[string]struct{}{
+	"openai":    {},
+	"anthropic": {},
+	"gemini":    {},
+}
+
+func isViewerPersonalProvider(provider string) bool {
+	_, ok := viewerPersonalProviders[strings.ToLower(strings.TrimSpace(provider))]
+	return ok
+}
+
+func isPersonalKey(key *apikeys.APIKey) bool {
```

golangci-lint (2.12.2)

[error] func isPersonalKey is unused

(unused)